epregressions/runtests.py

```diff
@@ -142,7 +142,7 @@ def run_test_suite(self):
         if self.id_like_to_stop_now:  # pragma: no cover
             self.my_cancelled()
             return
-        self.my_simulationscomplete()
+        self.my_simulations_complete()
```

Comment:
IDE was complaining about a bunch of these oddly spelled callbacks and such, so cleaned those up.  No functional change.